### PR TITLE
A smoother correction

### PR DESCRIPTION
**Pull request by eggrobin**
The thresholding toggle now switches between this behaviour when off and the Fubini behaviour + PID  (as in test2550) when on.

There seems to be an improvement: in the save given in https://github.com/mockingbirdnest/Principia/issues/2519#issuecomment-612605437, the vessel now explodes on reentry as it does in stock (the oscillations no longer slow it down enough for it to make it to the water unharmed).
However:
1. the oscillations on descent are still larger than in stock;
2. the westward drift on ascent is larger than in Fubini (where it was larger than in Frobenius).

Perhaps some PID tuning might help with those issues (especially 1).

### CODE REVIEW

File: ksp_plugin/pile_up.cpp [hunk 1]
```diff
@@ -578,20 +576,43 @@ void PileUp::DeformPileUpIfNeeded(Instant const& t) {
...
-  if (thresholding && α > ω * Δt) {
-    // The attitude correction is too large.  Preserve attitude.
-    apparent_pile_up_equivalent_rotation =
-        Rotation<ApparentPileUp, EquivalentRigidPileUp>::Identity();
-    actual_pile_up_equivalent_rotation =
-        Rotation<NonRotatingPileUp, EquivalentRigidPileUp>::Identity();
-    actual_equivalent_angular_velocity =
-        angular_momentum_ /
-        Identity<ApparentPileUp, NonRotatingPileUp>()(inertia_tensor);
+  // This correction angle is bounded by ωΔt (correct at most as fast as the
```
Comment (pleroy) [inline ksp_plugin/pile_up.cpp:588]: `ω * Δt`

File: ksp_plugin/pile_up.cpp [hunk 2]
```diff
@@ -578,20 +576,43 @@ void PileUp::DeformPileUpIfNeeded(Instant const& t) {
...
+    // |EquivalentRigidPileUp| were defined as in Fubini, with its y axis being
+    // L̂_apparent in |ApparentPileUp| and
+    //   Identity<ApparentPileUp, NonRotatingPileUp>()(
+    //       Rotation<ApparentPileUp, ApparentPileUp>(β)(L̂_apparent));
+    // in |NonRotatingPileUp|.  On the other hand, this would make the
+    // computations even more convoluted.
+    attitude_correction =
+        Rotation<ApparentPileUp, EquivalentRigidPileUp>::Identity() *
+        Rotation<ApparentPileUp, ApparentPileUp>(
+        β,
```
Comment (pleroy) [inline ksp_plugin/pile_up.cpp:610]: Bad indent (2x).

File: ksp_plugin/pile_up.cpp (hunk 1 above)
Comment (eggrobin) [reply ksp_plugin/pile_up.cpp:588]: ` `

**Comment by pleroy** [comment]
retest this please

**Pull request merged**